site/content/en/docs/Guides/fleet-updates.md

```diff
@@ -104,10 +105,39 @@ spec:
     - matchLabels:
         agones.dev/fleet: v2
 ```
+{{% /feature %}}
 
+{{% feature publishVersion="1.17.0" %}}
+```yaml
+apiVersion: "allocation.agones.dev/v1"
+kind: GameServerAllocation
+spec:
+# Deprecated, use field selectors instead.
```

Comment:
Owie my head hurts 😄 does this mean I need to use required and preferred and selectors all at the same time?

I think I get what you are going for here, but if we are deprecating `required` and `preferred`, I would probably advocate removing them from the guides -- or having two examples, once current, and one deprecated (maybe use a [docsy tabbed pane](https://www.docsy.dev/docs/adding-content/shortcodes/#tabbed-panes)???)